client/components/jetpack/upsell-switch/index.tsx

```diff
@@ -83,9 +99,11 @@ function UpsellSwitch( props: Props ): React.ReactElement {
 export default connect( ( state, ownProps: Props ) => {
 	const siteId = getSelectedSiteId( state );
 	const siteState = ownProps.getStateForSite( state, siteId );
+	const atomicSite = siteId ? isAtomicSite( state, siteId ) : false;
```

Comment:
I feel the code would be clearer with the conditional being `siteId && isAtomicSite( vars )`?

Reply:
Yes! better. I did it in the old-fashioned style :)